test/routes/user_spec.rb

```diff
@@ -1,8 +1,26 @@
+require 'pry'
 require_relative '../spec_helper.rb'
 require_relative '../../app.rb'
+require_relative '../../models/user.rb'
+require_relative '../../models/user_category.rb'
 
 RSpec.describe App  do
 
+  before(:context) do
+    User.delete_all
+    UserCategory.delete_all
+  end
+
+  it 'should add user user to the database' do
```

Comment:
the title of the test does not match what the test is doing - for ading something to the database, try `expect(User.count).to eq 1` or `expect { subject }.to change(User, :count).by(1)`